kibble/sample_site/templates/tv/detail.jet

```diff
@@ -33,3 +33,5 @@ Available To US West: [{{ tvseason.Available.To | zone: "America/Los_Angeles" |
     This is true
     {{ tvseason.CustomFields["string"] }} == {{ tvseason.CustomFields["number"] }}
 {{ end }}
+
+{{ stripHTML("tags <br>should be stripped") }}
```

Comment:
Just wondering what happens with `stripHTML("1 < 2 && 2 > 1")` or something where it's not an HTML tag but could look conceivably like one

Reply:
Works as expected: ` 1 &lt; 2 &amp;&amp; 2 &gt; 1`, I think the author [has it all pretty well covered](https://github.com/grokify/html-strip-tags-go/blob/master/strip_test.go)